updatecli/updatecli.d/plugin-compat-tester.yml

```
@@ -42,4 +42,4 @@ actions:
     title: Bump `plugin-compat-tester` version in prep.sh
```

```suggestion
    title: Bump `plugin-compat-tester` version in prep.sh to {{ source `pctVersion` }}
```
Add the version in the PR title